plasmapy/formulary/magnetostatics.py

```diff
@@ -250,14 +301,57 @@ def magnetic_field(self, p) -> u.T:
 
         Notes
         -----
-        Let :math:`P_f` be the foot of perpendicular, :math:`θ_1`
-        (\ :math:`θ_2`\ ) be the angles between :math:`\overrightarrow{PP_1}`
-        (\ :math:`\overrightarrow{PP_2}`\ ) and :math:`\overrightarrow{P_2P_1}`\ .
-
-        .. math:
-            \vec B = \frac{(\overrightarrow{P_2P_1}\times\overrightarrow{PP_f})^0}
-                     {|\overrightarrow{PP_f}|}
-                     \frac{μ_0 I}{4π} (\cos θ_1 - \cos θ_2)
+        The magnetic field generated by a straight, finite wire with
+        constant electric current can be found at a point in 3D space
+        using the Biot–Savart law.
+
+        Let the point where the magnetic field will be calculated be
+        represented by the point :math:`p_0` (or ``p``) and the wire's
+        beginning and end as :math:`p_1` and :math:`p_2`, respectively
+        (with corresponding position vectors :math:`\vec{p}_0`,
+        :math:`\vec{p}_1`, and :math:`\vec{p}_2`, respectively).
+        Further, the vector from points
+        :math:`p_i` to :math:`p_j` can be written as
+        :math:`\vec{p}_{ij} = \vec{p}_j - \vec{p}_i`.
+
+        Next, consider the triangle with the points :math:`p_0`,
+        :math:`p_1`, and :math:`p_2` as vertices. The foot of
+        perpendicular opposite the vertex :math:`p_0`, which will be
+        used to find the unit vector in the direction of the magnetic
+        field, can be expressed as
```

Comment:
I got a little confused here what "foot of perpendicular opposite the vertex" means. Would you be able to clarify or reword it?

Reply:
Considering the triangle with vertices `p0`, `p1`, and `p2`, ~~the "foot of perpendicular opposite the vertex `p0`"~~ the "vector from the vertex `p0` to the perpendicular foot opposite the vertex `p0`" (amended in 88e42da) refers to the vector from the vertex `p0` to the opposite side of the triangle (the line formed between `p1` and `p2`) and perpendicular to this opposite side.

I made a slight adjustment to the wording in 88e42da making it clearer to me, but let me know if it is still unclear for you. Thanks for looking over this.